.circleci/config.yml

```diff
@@ -0,0 +1,182 @@
+version: 2.1
+
+orbs:
+  win: circleci/windows@4.1
+
+jobs:
+  test_linux:
+    parameters:
+      ruby_version:
+        description: "version tag for the veracross/circleci-ruby-freetds container"
+        type: string
+
+    # Primary container image where all commands run
+    docker:
+      - image: cimg/ruby:<< parameters.ruby_version >>
+        environment:
+          TESTOPTS: "-v"
+          TINYTDS_UNIT_DATASERVER: sqlserver
+          TINYTDS_UNIT_HOST: sqlserver
+          SA_PASSWORD: super01S3cUr3
+          TOXIPROXY_HOST: toxiproxy
+
+      # Service container image
+      - image: mcr.microsoft.com/mssql/server:2019-CU16-ubuntu-20.04
+        name: sqlserver
+        environment:
+          ACCEPT_EULA: Y
+          SA_PASSWORD: super01S3cUr3
+
+      - image: shopify/toxiproxy:2.1.4
+        name: toxiproxy
+
+    working_directory: ~/repo
+
+    steps:
+      - checkout
+
+      - run:
+          name: setup sql
+          command: |
+            curl https://packages.microsoft.com/keys/microsoft.asc | sudo apt-key add -
+            curl https://packages.microsoft.com/config/ubuntu/20.04/prod.list | sudo tee /etc/apt/sources.list.d/msprod.list
+            sudo apt-get update
+            sudo ACCEPT_EULA=Y apt-get -y install mssql-tools unixodbc-dev
+            /opt/mssql-tools/bin/sqlcmd -S sqlserver -U sa -P $SA_PASSWORD -i ./test/sql/db-create.sql
+            /opt/mssql-tools/bin/sqlcmd -S sqlserver -U sa -P $SA_PASSWORD -i ./test/sql/db-login.sql
+
+      - run:
+          name: install dependencies
+          command: |
+            sudo ./test/bin/install-openssl.sh
+            sudo ./test/bin/install-freetds.sh
```

Comment:
Same comment here; it seems like this would be best as separate steps. I realize this is the equivalent of the "rake ports" step for windows, but I somewhat wish _that_ was split out too